# Scene object properties

int32     id                                  # Scene object's ID
std_msgs/String    mesh                       # Scene object's mesh
std_msgs/String    material                   # Scene object's material
float32[] location                            # Scene object's location
float32[] rotation                            # Scene object's rotation